tests/src/test/scala/system/basic/WskRestActionTests.scala

```diff
@@ -1,38 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package system.basic
-
-import common.{TestUtils, WskActorSystem}
-import common.rest.WskRestOperations
-import org.junit.runner.RunWith
-import org.scalatest.junit.JUnitRunner
-import spray.json._
-
-@RunWith(classOf[JUnitRunner])
-class WskRestActionTests extends WskActionTests with WskActorSystem {
-  override val wsk = new WskRestOperations
-
-  it should "create an action with an empty file" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
The docker skeleton and python runtimes accept an empty code string during init, and fail in run (docker skeleton will just run the stub, and python will return error = action did not return dictionary). 

Java will fail with class/method not found for main during class loader.

Swift will fail with did not generate an executable.

Dropping this test is OK - I don't think we're getting any increased coverage. It would be nice to make the failures uniform but this may be a little difficult for all the runtimes to normalize.

See #3839 and #3844.